### ISSUE TITLE
Save some precious vertical space and screen real estate by combining the lines of some feed item metadata in the previewer pane

### ISSUE THREAD

**Issue by nekohayo** [lwindolf/liferea#911]
Here I will provide two pictures that will speak louder than a thousand words. It's a little annoyance that has been bugging me for years, with zones of wasted UI real estate colored in red, orange and yellow (excuse the grotesque font, it's just because it's stands out better from the background that way):

![wasted space in the feed item header in the previewer](https://user-images.githubusercontent.com/479401/99693892-dc374e00-2a59-11eb-9d10-cce60cd6a000.png)

...and here's my proposed solution:

![wasted space in the feed item header in the previewer - mockup](https://user-images.githubusercontent.com/479401/99693921-e0fc0200-2a59-11eb-9104-636bf62115a7.png)

It boils down to making most of the metadata fit as a single translatable string, roughly this: 

`"By <b>$AUTHOR</b>, from <b><a href="foo">blog/website URL</a></b> — filed under <i>the tags and categories</i>"`

It would look 120% sexier AND it would consistently save 66% of the surface area's space AND it would reduce scrolling for short item contents. In terms of pixels real estate, on a 1280px-wide laptop screen (or a Liferea window, or a wide-layout Liferea on an ultrawide display, I suppose), my proposal makes it go from eating roughly 60,000 pixels² (1000x60) to 20,000 pixels² (1000x20).

**Comment by mikelolasagasti** [comment]
This should be changed in https://github.com/lwindolf/liferea/blob/master/xslt/item.xml.in iirc, I can have a look.

**Comment by nekohayo** [comment]
> joining a separator like "---" is not easy

How do you mean? Isn't it just having that unicode character there in the string?

>  Also LTR/RTL scenarios could make troubles.

The way I see it, the entire line could be one translatable string with variables in it. There's probably a way to do something like [Python's f-strings](https://docs.python.org/3/tutorial/inputoutput.html) but I don't know what's the C equivalent?

If it was all in one big translatable string, translators would have the ultimate control over the ordering of its items, and I would expect it to just work in RTL languages because of that?

**Comment by lwindolf** [comment]
YOu are right it would work. But I do not like it from an architectural standpoint. The code base does use XSLT+gettext for translating and Handlebars.js for layout templating. So putting layout stuff in the translation is not a good idea. Having to update all translations just to change the order of elements in this line shows that it is not a good idea.

Handlebars.js itself is a template engine which knows no such things are procedural concatenation. It is used for security reasons to 100% enforce HTML escaping. Just "concatenating" stuff opens up to malicious content injection again.
